sources/packages/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -15,13 +15,16 @@ import { IUserToken } from "../../auth/userToken.interface";
 import { transformToNoteDTO, transformToNoteEntity } from "../../utilities";
 import { ApiTags } from "@nestjs/swagger";
 import { Role } from "../../auth/roles.enum";
+import { ClientTypeBaseRoute } from "../../types";
 /**
  * Controller for Notes.
  * This consists of all Rest APIs for notes.
  */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
 @Controller("notes")
-@ApiTags("notes")
-export class NotesController extends BaseController {
+@ApiTags(`${ClientTypeBaseRoute.AEST}-notes`)
+export class NotesAESTController extends BaseController {
```

Comment:
The controller is not doing the usual checks to throw `NotFoundException`, for instance.